getFullStructs.m

```diff
@@ -0,0 +1,108 @@
+function [parameters,properties] = getFullStructs(parameters,reducedParameters,properties,reducedProperties,options)
```

Comment:
Hard to deduce what this function does from the name alone.

Reply:
General comment on this function:

This function is not used in the current code version. Jan started working on it in #121 to restructure the way how fixed parameters are currently handled.
However, I think this PR fixes enough issues already and we  should not include these files right now, even if we agreed on using them later. I suggest leaving the current implementation for fixed parameters for this PR.
Any objections to this idea?